scripts/variantstore/wdl/GvsExtractCallsetPgen.wdl

```diff
@@ -130,22 +130,20 @@ workflow GvsExtractCallsetPgen {
     }
 
     Int effective_scatter_count = if defined(scatter_count) then select_first([scatter_count])
-                                  else if GetNumSamplesLoaded.num_samples < 100 then 100 # Quickstart
-                                       else if GetNumSamplesLoaded.num_samples < 1000 then 500
-                                            else if GetNumSamplesLoaded.num_samples < 5000 then 1000
-                                                 else if GetNumSamplesLoaded.num_samples < 20000 then 2000 # Stroke Anderson
-                                                      else if GetNumSamplesLoaded.num_samples < 50000 then 10000
-                                                           else if GetNumSamplesLoaded.num_samples < 100000 then 20000 # Charlie
-                                                                 else 34000
+                                  else if GetNumSamplesLoaded.num_samples < 5000 then 1
```

Comment:
is this intentional?

Reply:
see https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/wdl/GvsExtractCallsetPgen.wdl#L132

Reply:
PGEN extract doesn't work exactly like VCF extract, there's a separate post stage to merge PGEN shards by chromosome. Has this actually been run at < 5000 scale? I'm not sure the way the PGEN extract shards are configured now that they would survive scattering so narrowly.

Reply:
The test run linked to at the top of this PR is quickstart data (10 samples).